orchagent/intfsorch.cpp

```diff
@@ -833,6 +863,14 @@ void IntfsOrch::doTask(Consumer &consumer)
                         }
                         gPortsOrch->setPort(alias, port);
                     }
+                    /* Set MPLS */
+                    if ((!ip_prefix_in_key) && (port.m_mpls != mpls))
```

Comment:
why not handle case where IP prefix is in the key.

Reply:
@smaheshm In INTERFACE stanza, mpls enable/disable attr belongs in single-key (name only) entry, similar to vrf_name attr.  So this code looks for mpls attr only in single-key entry, not double-key (name+ip prefix) entry.